### PR TITLE
I18n sync Down & Out 10/30

### PR DESCRIPTION
**Pull request by deploy-code-org**
# I18n Sync Down & Out

This PR contains all changes to translations made since the last sync (usually a week).

## To Review:

Look through the changes in each of the commits and verify that all changes are expected. Note that **it is not necessary to review every single "dashboard i18n update" commit.** These commits are batched by language to make it easier to review just a couple commits in this category.

### Things to Note:

- We expect translation changes to come in on a regular basis; new strings will be translated, existing translations will be updated, and source string removals will result in translations for those strings also being removed.

- We expect translations to arrive in large batches, so don't worry if you see a large group of changes in a specific language.

- We _do not_ usually expect to see a large number of formatting changes, or batches of changes that apply to strings in every single language. If you see this, it usually means either that a code change went in or that something broke.

## Known Issues:

- If the CI unit tests fail at the `grunt messages:all` step, they will typically fail with a message like:

  ```
  [2019-10-04 02:38:57] Running "messages:all" (messages) task
  [2019-10-04 02:39:00] Warning: Error processing localization file i18n/common/zh_tw.json: SyntaxError: Expected [0-9a-zA-Z$_] but "\u8A0A" found. Use --force to continue. 
  ```

  This usually means that a translator has translated one of the [MessageFormat variables](https://messageformat.github.io/messageformat/guide/#variables). In this case, you should find and fix the offending translation in crowdin, and then manually update the associated strings in `i18n/locales` and `apps/i18n`.

  See https://github.com/code-dot-org/code-dot-org/pull/31098/commits/ab3e5d1d740869a880f2e9da5cf0a4348886915e and https://github.com/code-dot-org/code-dot-org/pull/31098/commits/0fca37658cacaaff55d9adfc9d04c40b37cc5e2d for an example.

## To Deploy:

Once one or two people have reviewed and approved this change and the tests are passing, ship it!

It's not necessary to wait for everyone on the team to review.

### CODE REVIEW

**Comment by deploy-code-org** [comment]
This comment is created by running: `bin/i18n/analyze_pr.rb -p 54566 -c`

Top 5/192 files sorted by the number of changes:
- `bin/i18n/crowdin/hour-of-code_etags.json`, 4156 changes (2078 additions, 2078 deletions).
- `dashboard/config/locales/behavior_descriptions.ar-SA.json`, 526 changes (263 additions, 263 deletions).
- `i18n/locales/vi-VN/hourofcode/vi.yml`, 333 changes (161 additions, 172 deletions).
- `i18n/locales/te-IN/hourofcode/te.yml`, 326 changes (159 additions, 167 deletions).
- `i18n/locales/uk-UA/hourofcode/uk.yml`, 309 changes (152 additions, 157 deletions).

Top 5/13 commits sorted by the number of changes:
- Commit: hoc i18n updates
  - 33614 changes (16662 additions, 16952 deletions).
  - 139 files changed.
  - File with the most changes: `i18n/locales/vi-VN/hourofcode/vi.yml`, 333 changes (161 additions, 172 deletions).
- Commit: etags updates
  - 4288 changes (2144 additions, 2144 deletions).
  - 3 files changed.
  - File with the most changes: `bin/i18n/crowdin/hour-of-code_etags.json`, 4156 changes (2078 additions, 2078 deletions).
- Commit: dashboard i18n updates - Arabic
  - 750 changes (380 additions, 370 deletions).
  - 13 files changed.
  - File with the most changes: `dashboard/config/locales/behavior_descriptions.ar-SA.json`, 526 changes (263 additions, 263 deletions).
- Commit: apps i18n updates
  - 340 changes (244 additions, 96 deletions).
  - 12 files changed.
  - File with the most changes: `apps/i18n/common/fa_ir.json`, 67 changes (56 additions, 11 deletions).
- Commit: dashboard i18n updates - Polish
  - 268 changes (254 additions, 14 deletions).
  - 7 files changed.
  - File with the most changes: `dashboard/config/locales/activity_sections.pl-PL.json`, 207 changes (196 additions, 11 deletions).

File: dashboard/config/locales/resources.th-TH.json [hunk 1]
```diff
@@ -7971,7 +7974,7 @@
         },
         "classifying_fish/how-ai-works/2023": {
           "name": "การจำแนกปลา",
-          "url": "https://docs.google.com/document/d/1gQWtamQ3YBk-gJnmw_z3QWNLfzdrPyaUjS4ZbtfAvEs/view",
+          "url": "https://docs.google.com/document/d/1-jk6nypSNHxokQP3DJRgITQdvjkNGFzNsP2wYM9vfeg/",
```
Comment (mgc1194) [inline dashboard/config/locales/resources.th-TH.json:7977]: @castro-jorge flagging these links to resources. Please double-check these updates are expected. I have no access to the documents, and I would assume students/teachers won't either.

File: i18n/locales/th-TH/hourofcode/th.yml [hunk 2]
```diff
@@ -241,20 +243,21 @@ th:
...
   call_to_action_register_your_hoc: ลงทะเบียน Hour of Code ของคุณ
   call_to_action_select_activity: เลือกกิจกรรม
   call_to_action_start_activity: เริ่มกิจกรรม
   call_to_action_get_certificates: รับใบรับรอง
   call_to_action_go_beyond_hoc: ไปให้ไกลกว่า Hour of Code
-  call_to_action_explore_all_hoc_activities: สำรวจกิจกรรมHour of Code ทั้งหมดจาก Code.org
-  call_to_action_start_dance_party: เริ่มปาร์ตี้เต้นรำ
-  call_to_action_start_part_02: เริ่มตอนที่สอง
-  call_to_action_start_unplugged_activity: เริ่มกิจกรรมที่ถอดปลั๊ก
+  call_to_action_explore_all_hoc_activities: Explore all Hour of Code activities from
```
Comment (mgc1194) [inline i18n/locales/th-TH/hourofcode/th.yml:256]: This strings are regressing to English :(

Comment (mgc1194) [reply i18n/locales/th-TH/hourofcode/th.yml:256]: Never mind, I saw the translations were actually canceled in the Crowdin project. There are already new translations that will be picked up in the next sync.

**Review by mgc1194 — APPROVED** [review]

File: dashboard/config/locales/resources.th-TH.json (hunk 1 above)
Comment (castro-jorge) [reply dashboard/config/locales/resources.th-TH.json:7977]: @mgc1194 our Turkish partner updated the share settings of all the documents.

**Pull request merged**